src/examples/colors/generate_wcs_languages.py

```diff
@@ -222,69 +239,95 @@ def meaning_distance(center, point):
 
         color_names = temp_color_terms
 
-        #Create a zeroed np array of the number of colors x the number of major color terms within the language
-        expression_meanings = np.array([[0.0 for _ in range(len(color_universe.referents))] for _ in range(len(major_color_terms))])
+        # Create a zeroed np array of the number of colors x the number of major color terms within the language
+        expression_meanings = np.array(
+            [
+                [0.0 for _ in range(len(color_universe.referents))]
+                for _ in range(len(major_color_terms))
+            ]
+        )
 
-        #Print the expression array dimensions
-        #print(f"Expression array dimensions: {expression_meanings.shape}")
+        # Print the expression array dimensions
+        # print(f"Expression array dimensions: {expression_meanings.shape}")
 
-        #For each major color chip in the language, add the Gaussian meaning row corresponding to that chip to the row for that color term.
+        # For each major color chip in the language, add the Gaussian meaning row corresponding to that chip to the row for that color term.
         for color in language_colors:
-            color_meaning_dist = meaning_dists[list(color_universe.referents).index(referents_by_color_code[color])]
-            for major_term, major_term_count in language_colors_to_expressions[language_code][color].items():
+            color_meaning_dist = meaning_dists[
+                list(color_universe.referents).index(referents_by_color_code[color])
```

Comment:
putting the `list` inside here is very expensive since it will be edone every time

Reply:
I'll instantiate a list once when the universe is created